apps/modernization-ui/src/design-system/checkbox/checkboxGroup.module.scss

```diff
@@ -1,36 +1,22 @@
-@use 'styles/colors';
-@use 'styles/components';
-
 .checkboxGroup {
-    padding: 0;
-    border: none;
-
-    legend {
-        padding-top: 0;
-        color: colors.$base-darkest;
-        margin-bottom: 0.5rem;
-    }
-
-    .error {
-        margin-bottom: 0.5rem;
-        padding-top: 0;
+    &.basic-race-ethnicity {
```

Comment:
issue:  If this is specific to the basic race and ethnicity form it should be specified there and not in a core component.  Are these styles really specific to this section or actually styles of a horizontal checkbox group.?